drivers/hmis/app/models/hmis/ce/match/cde_field_map.rb

```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+# resolve custom data elements (CDEs)
+module Hmis::Ce::Match
+  class CdeFieldMap
+    def instance_value(client, field)
```

Comment:
this could be made more efficient. Wanted to confirm it is logically correct before optimizing